frontend/netlify.toml

```diff
@@ -0,0 +1,10 @@
+[build]
+  ignore = "git diff --quiet HEAD^ HEAD -- '*.ts*' '*.js*' 'yarn.lock'"
```

Comment:
hmm this returns exit code `1` for me on a clean repo, which i think would trigger a build?

Reply:
It's running against the previous commit. Can you check if the prior commit on your checkout has ts or lockfile changes in it?

Reply:
Ah makes sense.

For PRs is it evaluating a merge commit? Because we need to diff against main not commits within the PR itself.

Reply:
Last thing this will match .json in addition to .js and .jsx, not sure if intentional.

Reply:
Oh good point. It would evaluate against the previous commit on the PR. I can update this to use main as the base for comparison

Reply:
json is intentional since we might make changes to package.json for storybook builds